ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -7,7 +7,6 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_method
 from ivy.functional.frontends.paddle import Tensor
 
-
```

Comment:
nit: please undo these changes; they are not related to the PR scope, nor are they required.